solana/modules/token_bridge/program/src/instructions.rs

```diff
@@ -1,3 +1,4 @@
+#![allow(clippy::too_many_arguments)]
```

Comment:
If there aren't a lot of functions where this applies we can put this right next to those functions rather than allowing it for the entire crate.